src/selectors/selectSourceControlTabProps.ts

```diff
@@ -71,23 +71,43 @@ export const selectSourceControlTabProps = (
 	const job = state.job.entities[sourceControlState.jobHash] ?? null;
 
 	if (job === null) {
-		throw new Error('Unable to get the job');
+		return null;
 	}
 
-	const { beforeSnippet, afterSnippet } = createBeforeAfterSnippets(
+	const title = `[Codemod:${job.codemodName}] Invalid codemod output`;
+	const { beforeSnippet, afterSnippet: newFileSnippet } =
+		createBeforeAfterSnippets(
+			sourceControlState.oldFileContent,
+			sourceControlState.newFileContent,
+		);
+
+	if (sourceControlState.modifiedFileContent === null) {
```

Comment:
why this `if`?

Reply:
`sourceControlState.modifiedFileContent === null` means that user didn't modify the file. In this case, we don't need to do the following

```
const { afterSnippet: modifiedFileSnippet } = createBeforeAfterSnippets(
		sourceControlState.oldFileContent,
		sourceControlState.modifiedFileContent,
	);
```